docs/introduction.md

```diff
@@ -0,0 +1,20 @@
+# Introduction
```

Comment:
Rename this file to `README.md`, in such a way that GitHub will show it by default when opening the containing folder

Reply:
Since it will be the home page, add a link to the list (current `documentation.md` file).